enacts/onset/tests/test_calc.py

```diff
@@ -169,13 +177,28 @@ def test_water_balance2():
     ]
     precip = xr.DataArray(values, dims=["X", "T"], coords={"T": t})
     wb = calc.water_balance(precip, 5, 60, 0)
+    assert np.array_equal(wb.soil_moisture["T"][1:], t)
+    expected = [
+        [0.0, 0.0, 1.0, 0.0, 60.0],
+        [0.0, 5.0, 12.0, 21.0, 32.0],
+    ]
+    assert np.array_equal(wb.soil_moisture.transpose(), expected)
+
+
+def test_water_balance2_reduce_True():
 
-    assert np.array_equal(wb.soil_moisture["T"], t)
+    t = pd.date_range(start="2000-05-01", end="2000-05-04", freq="1D")
+    values = [
+        [5.0, 6.0, 3.0, 66.0],
+        [10.0, 12.0, 14.0, 16.0],
+    ]
+    precip = xr.DataArray(values, dims=["X", "T"], coords={"T": t})
+    wb = calc.water_balance(precip, 5, 60, 0, reduce=True)
     expected = [
-        [0.0, 1.0, 0.0, 60.0],
-        [5.0, 12.0, 21.0, 32.0],
+        [60.0],
+        [32.0],
     ]
-    assert np.array_equal(wb.soil_moisture, expected)
+    assert np.array_equal(wb.soil_moisture.transpose(), expected)
```

Comment:
What's going on with this `transpose`? Why not just write `expected` to be what you're actually expecting?

Reply:
just being lazy

Reply:
If it's more convenient to write the transposed version, then you could do
```
expected = [ ... ].transpose()
```
Less confusing without being less lazy.